terraso_backend/apps/core/models/users.py

```diff
@@ -114,6 +115,33 @@ def soft_delete_policy_action(self, **kwargs):
     def is_group_manager(self, group_id):
         return self.memberships.managers_only().filter(group__pk=group_id).exists()
 
+    def full_name(self):
+        return f"{self.first_name} {self.last_name}".strip()
+
+    def notifications_enabled(self):
+        preferences = self.preferences.filter(key="notifications")
+        if len(preferences) != 1:
+            return False
+
+        try:
+            return bool(preferences[0].value)
```

Comment:
suggestion: the Python bool conversion only returns `False` for an empty string and `True`. If that's not what we want, we can do something like the following:

```suggestion
            return preferences[0].value.lower() == "true"
```

Reply:
Not sure what we storing in the DB yet — will update to match your suggestion once that is settled.